<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CEditValueMapDialog</class>
 <widget class="QDialog" name="CEditValueMapDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数值映射表属性</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="5,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,7">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>名称:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineeditName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,7">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>说明:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineeditDesc"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="CValueMapView" name="tableviewMap"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushbuttonClearAll">
       <property name="text">
        <string>清空表格数据</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushbuttonAdd">
       <property name="text">
        <string>增加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushbuttonDelete">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushbuttonOk">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushbuttonCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushbuttonDeleteMap">
       <property name="text">
        <string>删除该表并退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CValueMapView</class>
   <extends>QTableView</extends>
   <header>ValueMap/ValueMapView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
